package.json

```diff
@@ -181,6 +203,7 @@
     "redux-thunk": "2.3.0",
     "refx": "3.1.1",
     "rememo": "3.0.0",
+    "request": "2.88.0",
```

Comment:
Do we use the `request` dependency anywhere?

Reply:
The dependency (`useragent`) for one of our dependencies (`browserslist-useragent`) requires this to be specified if certain functionality is used, which our dependency has chosen to do. See https://www.npmjs.com/package/useragent#api

This was breaking the desktop build. I can try with just `yamlparser` and remove `request`, though.

Reply:
Submitted a commit to remove `request` as a direct dependency. We'll see how it goes!

Reply:
Hmm, so the `useragent` package optionally makes network requests during the build? I'd rather avoid that and have the build completely offline.

Until now, making network requests was an exclusive property of evil packages. E.g., `eslint-scope` that stealed your NPM credentials or `event-stream` that stealed your Bitcoins. Let's keep it that way 🙂

One day we might want to sandbox our builds and prevent them from making any network requests other than the NPM registry (and even that can be possibly cached).

If the `caniuse-db` package and the feature support database in `@babel/preset-env` can be offline, then a list of UA regexp can be offline, too.

Reply:
Those are some very good points.

I dig into this a bit deeper. As far as I can tell, there appears to be nothing in `browserslist-useragent` that would trigger a network request, since `useragent` isn't being initialised as the documentation link I provided states it should be.

My guess is that `yamlparser` is missing as an explicit dependency, and `request` is not being used at all. We'll know for sure once the current desktop build finishes.

Reply:
Confirmed, only `yamlparser` is needed. Build-time network requests avoided :)